### ISSUE TITLE
Useless condition

### ISSUE THREAD

**Issue by hadware** [IEMLdev/ieml#54]
https://github.com/IEMLdev/ieml/blob/ac1f9ee54f290a746db6e15396694de1b0f79352/ieml/grammar/parser/parser.py#L131

I think this condition was used for closed propositions, but can now be remove.

**Comment by ogrergo** [comment]
I agree ! but look at the dev branch, it is the one that is up to date and used in other application. I will soon merge it to master. In dev, you will see two parsers in 
ieml/ieml/lexicon/grammar/parser/ : the parser2 folder contains the parser for the lexical level with the new grammar

**Comment by hadware** [comment]
Ah right sorry
